### ISSUE TITLE
Polish the UI. "Make it pop!"

### ISSUE THREAD

**Issue by damms005** [damms005/devdb-vscode#14]
Nice smooth UI animations, cool use of soft-shadows, etc. Software developement is hard enough: DevDB should provide a delightful experience to developers.

Case studies
- https://youtu.be/0_2xORiONDo

Possible UI libs:
- https://primevue.org
- https://vuesax.com/docs/components

**Comment by damms005** [comment]
Inspiration...


![20241026_144055.jpg](https://github.com/user-attachments/assets/915db058-eb21-4523-9689-257b4b2273c0)

**Comment by nsrosenqvist** [comment]
I'd prefer if the UI was styled the same as the VS Code theme. Is there a way to make sure the theme styling gets applied instead of the bundled style?

**Comment by damms005** [comment]
Nah. It is still Day 0 for DevDb. The plans ahead are huge and it will be a lot of unnecessary boilerplate to style based on IDE theme.

That said, there will always be light/dark variants of the bundled style, which should at least suffice for the light/dark variants of IDE themes.
